cherab/tools/observers/bolometry.py

```diff
@@ -127,14 +128,16 @@ def observe(self):
 
 class BolometerSlit(Node):
 
-    def __init__(self, slit_id, centre_point, basis_x, dx, basis_y, dy, dz=0.001, parent=None, csg_aperture=False):
+    def __init__(self, slit_id, centre_point, basis_x, dx, basis_y, dy, dz=0.001,
+                 parent=None, csg_aperture=False, curvature_radius=0):
 
         self._centre_point = centre_point
         self._basis_x = basis_x.normalise()
         self.dx = dx
         self._basis_y = basis_y.normalise()
         self.dy = dy
         self.dz = dz
+        self._curvature_radius = curvature_radius
```

Comment:
You don't have any validation on the curvature_radius variable in the BolometerSlit. But you do have it already for BolometerFoil. Please copy over.